micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/AbstractTimeWindowHistogram.java

```diff
@@ -54,9 +55,11 @@ abstract class AbstractTimeWindowHistogram<T, U> implements Histogram {
     private volatile long lastRotateTimestampMillis;
 
     @SuppressWarnings({ "unused", "FieldCanBeLocal" })
-    private volatile int rotating; // 0 - not rotating, 1 - rotating
+    private volatile int rotating;
 
-    @Nullable
+    // 0 - not rotating, 1 - rotating
```

Comment:
I don't know why this comment got moved to another line, but we should keep it the same as before.